tokens/Cargo.toml

```diff
@@ -1,43 +1,40 @@
 [package]
-name = "orml-tokens"
+authors     = ["Laminar Developers <hello@laminar.one>"]
```

Comment:
Please revert all your format change in `toml` files and keep the previous format and orders.

Reply:
"*and orders*"

Reply:
ok I'll change it